### DISCUSSION TITLE
Possible to add an optional menu?

### DISCUSSION THREAD

**Discussion by anthrolisp** [maximevaillancourt/digital-garden-jekyll-template#77] [Ideas]
Is it possible to add an optional menu to this template? Thanks!

**Comment by florhizome** [comment]
I just implemented the sidebar of the “Lanyon” theme for myself, it’s pretty simple when you’re halfway into Jekyll’s framework. Could be added as a submodule, maybe. Or if @maximevaillancourt opens a wiki ppl could publish their tweaks/usage off the theme allowing for the template and maintenance to remain minimal.
